static/src/js/actions/portals.js

```diff
@@ -14,12 +15,17 @@ export const addPortal = (payload) => ({
  * Loads the portal config into the Redux Store.
  * @param {String} portalId Portal Name, must match the directory name in which the config is stored.
  */
-export const loadPortalConfig = (portalId) => (dispatch) => {
-  if (!portalId) return
+// eslint-disable-next-line max-len
```

Comment:
Don't disable max-len for this, move `portalId = getApplicationConfig().defaultPortal` onto it's own line